### ISSUE TITLE
One variant per gene

### ISSUE THREAD

**Issue by snesic** [broadinstitute/tensorqtl#8]
Hi, 

Here is a code that returns an error (too many indices for tensor of dimension 0) when there is only one cis-variant. The error appears when you uncomment gt_pos line.  

```
import pandas as pd
from tensorqtl import cis
from numpy.random import randint

samples = 10
snps = 2
genes = 2

gt = pd.DataFrame(randint(-1, 3, size=(snps,samples)), 
                  columns=['sample_{}'.format(i) for i in range(1,samples+1)], 
                  index=['snp_{}'.format(i) for i in range(1,snps+1)])

gt_pos = pd.DataFrame({'pos' : list(range(1,snps+1)),
                      'chrom': '1'},
                      index = gt.index)

#gt_pos.loc['snp_2','chrom'] = '2'

ph = pd.DataFrame(randint(5, 15, size=(genes, samples)), 
                  columns = gt.columns, 
                  index = ['gene_{}'.format(i) for i in range(1,genes+1)])

ph_pos = pd.DataFrame({'chr' : ['1' ,'1'], 'tss' : [100,1000]})
ph_pos.index = ph.index

covariates_df = pd.DataFrame(1, index=['C'], columns=ph.columns).T

cis_df = cis.map_cis(gt, gt_pos, ph, ph_pos, covariates_df)
```

Thanks

**Comment by francois-a** [comment]
Thanks for the reproducible example and for catching this! Fixed in 026d394d09f529d151086a60d5c815efa7f850e1.

**Comment by snesic** [comment]
Excellent, thanks!!
